.flowconfig

```diff
@@ -1,9 +1,10 @@
 [ignore]
-.*/lib/*
+lib/*
```

Comment:
Neither of these are correct actually 😛

This should be either `.*/lib/.*` or `<PROJECT_ROOT>/lib/.*` according to [the docs](https://flowtype.org/docs/advanced-configuration.html#ignore).